src/server/pkg/serviceenv/option.go

```diff
@@ -1,24 +1,21 @@
 package serviceenv
 
 import (
-	"os"
 	"path/filepath"
 
 	"github.com/pachyderm/pachyderm/src/server/pkg/obj"
 	"github.com/pachyderm/pachyderm/src/server/pkg/storage/chunk"
 	"github.com/pachyderm/pachyderm/src/server/pkg/storage/fileset"
 )
 
-var localDiskCachePath = filepath.Join(os.TempDir(), "pfs-cache")
-
 // ChunkStorageOptions returns the chunk storage options for the service environment.
 func (env *ServiceEnv) ChunkStorageOptions() ([]chunk.StorageOption, error) {
 	var opts []chunk.StorageOption
 	if env.StorageUploadConcurrencyLimit > 0 {
 		opts = append(opts, chunk.WithMaxConcurrentObjects(0, env.StorageUploadConcurrencyLimit))
 	}
 	if env.StorageDiskCacheSize > 0 {
-		diskCache, err := obj.NewLocalClient(localDiskCachePath)
+		diskCache, err := obj.NewLocalClient(filepath.Join(env.CacheRoot, "pfs-cache"))
```

Comment:
This is the V2 code path, which isn't exercised in the 1.13.x branch, so this isn't really necessary but it is fine to leave it. We have a PR to remove this code, we just haven't merged it in.